prestodb/hdp2.6-hive-kerberized/files/etc/supervisord.d/kdc.conf

```diff
@@ -1,13 +1,13 @@
 [program:krb5kdc]
-command=/bin/bash -c "exec /usr/sbin/krb5kdc -r LABS.TERADATA.COM -P /var/run/krb5kdc.pid -n"
+command=/bin/bash -c "exec /usr/sbin/krb5kdc -P /var/run/krb5kdc.pid -n -r LABS.TERADATA.COM -n -d /var/kerberos/krb5kdc/principal-other -r OTHERLABS.TERADATA.COM"
```

Comment:
nit: you can remove `/bin/bash -c "` and final `"`

as i understand this runs single KDC process for the two realms. Should we have two processes to better simulate multi-realm world?